### ISSUE TITLE
0.3.0 Endgame

### ISSUE THREAD

**Issue by Eskibear** [Microsoft/vscode-spring-initializr#57]
### Dev
- [x] Remove dependency of `unzip` package. @Eskibear
- [x] Support to edit starters for existing Spring Boot Maven projects. @Eskibear 
- [x] Allow not to select extra dependencies when generate projects. @Eskibear 

### Test
- [x] 4/28 New feature test.
- [x] 5/2 Releasing test, following [Test Plan](https://github.com/Microsoft/vscode-spring-initializr/wiki/Test-Plan)
### Release
- [x] 5/4 Update Readme & changelog 
- [x] 5/4 Merge to master
- [x] 5/4 Trigger release job
- [x] 5/4 Add tag

**Comment by bsaby** [comment]
# New feature test
1.  Open Bug
#58 #59 #60 
2. Test result

Test case | Windows | OSX | Linux
-- | -- | -- | --
Owner | Mona | Sally | Mona
Clean   install | Pass | Pass | Pass
Update | Pass | Pass | Pass
Uninstall | Pass | Pass | Pass
The pom.xml file   is modified, and the project can still be built by mvn package. | #59 | #59 | #59
Corresponding <dependency> nodes   are added/removed. | #58 | #58 | #58
For   dependencies with bom information, e.g. azure,   a <DependencyManagement> node is added into the pom file. | Pass | Pass | Pass
For   dependencies with repository information, e.g. Spring Shell,   a <repository> node is added into the pom file. | Pass | Pass | Pass

**Comment by Eskibear** [comment]
#59 was fixed in #62

**Comment by Eskibear** [comment]
Won't fix the unblocking issues (#58 #60 )  in this release.

**Comment by bsaby** [comment]
# 5/2 Releasing test
1. Open bug: #64 
2. Test result:

Test case | Windows | OSX | Linux
-- | -- | -- | --
Clean   install | Pass | Pass | Pass
Update | Pass | Pass | Pass
Uninstall | Pass | Pass | Pass
Generate   a Maven Project: Quickstart   with dependency search | Pass | Pass | Pass
Generate   a Maven Project:Quickstart with last settings | Pass | Pass | Pass
Generate a Gradle Project: Quickstart with dependency search | Pass | Pass | Pass
Generate   a Gradle Project: Quickstart   with last settings | Pass | Pass | Pass
Customized   Spring Initializr Service URL | Pass | Pass | Pass
Default value of   GroupId and ArtifactId | Pass | Pass | Pass
Edit Starters   for Spring Boot Maven projects | #64 | #64 | #64
Can   generate Kotlin and Groovy projects | Pass | Pass | Pass

**Comment by Eskibear** [comment]
v0.3.0 released.
